Dockerfile.ubi9

```diff
@@ -1,4 +1,11 @@
 # syntax=docker/dockerfile:1
+#---------------------------------------------------------------------
+# PRE-BUILD STAGE: Build the ecr credential helper binary
```

Comment:
it is the `acr` cred helper that we're building, and not the `ecr`, right?

Reply:
Oops, got them mixed up in the comments! thanks!